app/utils/Utils.ts

```diff
@@ -19,7 +19,8 @@ export const createRules = (rules: any): any => {
   return rules;
 };
 
-export const openUrl = (url: string, newTab = true): void => {
+export const openUrl = (url: string | undefined, newTab = true): void => {
+  if (!url) return;
```

Comment:
As far as I can see openUrl is never called with `undefined` (there is a `user.bsLink &&  (...)`)